core/src/components/cat-input/cat-input.tsx

```diff
@@ -268,6 +268,11 @@ export class CatInput {
    */
   @Event() catBlur!: EventEmitter<FocusEvent>;
 
+  /**
+   * Emitted if the input type is "file" and files are selected.
+   */
+  @Event() filesSelected!: EventEmitter<FileList | null>;
```

Comment:
OCD kicking in, but can we rename this to something like `catChangeFiles` to stay in line with our current event naming pattern?